spec/models/collection_spec.rb

```diff
@@ -37,26 +29,15 @@
       let(:work1) { create_for_repository(:work) }
       let(:work2) { create_for_repository(:work) }
 
-      it "allows multiple files to be added" do
+      before do
         collection.add_member_objects [work1.id, work2.id]
-        collection.save!
-        expect(collection.reload.member_objects).to match_array [work1, work2]
+        persister.save(resource: collection)
       end
-    end
-  end
 
-  describe "#destroy", clean_repo: true do
```

Comment:
Can you explain why testing ` #destroy` is being removed?

Reply:
Because in Valkyrie you don't destroy on a model.  You destroy on a change set.